compiler/one-cmds/onnx_legalizer.py

```diff
@@ -0,0 +1,749 @@
+#!/usr/bin/python3
+import onnx
+import onnx.numpy_helper
+import sys
+import numpy as np
+import re
+
+# Transform onnx model to make it compilable with our toolchain
+# List of transformations:
+# - Replace RNN operation with unrolled subgraph
+# - Replace LSTM operation with unrolled subgraph
+
+
+def reverse_str(s):
+    return ''.join(reversed(s))
+
+
+def parse_tensor_name(name):
```

Comment:
It has nothing to do with this PR. Just like C++ private attribute, I've written methods whose name starts with "_". But, it turns out that underscore seems unnecessary and doesn't look good to me. So, I'm gonna post a PR that removes them. What is your opinion about this?

Reply:
> underscore seems unnecessary and doesn't look good to me

As far as I know, underscore in function names affects `imports * from module` expressions.
Sooo, maybe I need to add underscores to some functions here =)

P.s. I personally think that this code needs some refactoring, but deadlines are near, so published it as-is =_=
Sorry, if it is hard to read...